tests/dags/conftest.py

```diff
@@ -8,19 +8,41 @@
 """
     Provide teh necessary Airflow magic so that it is easy to test the production DAGs.
 """
+import json
 import logging
 import os
+import pathlib
 from pathlib import Path
+from typing import List
 
 import pytest
+from SPARQLWrapper import SPARQLWrapper
 from airflow.models import DagBag
 from airflow.utils import db
 
 SRC_AIRFLOW_DAG_FOLDER = Path(__file__).parent.parent.parent / "sem_covid/"
+TEST_DATA_FOLDER = pathlib.Path(__file__).parent.parent / "test_data"
 
 logger = logging.getLogger(__name__)
 
 
+class FakeSPARQL(SPARQLWrapper):
```

Comment:
Let's investigate where this is used and remove those usages, replacing them with our Triple Store adapter.